### ISSUE TITLE
wrap tangram related options as an object

### ISSUE THREAD

**Issue by hanbyul-here** [mapzen/mapzen.js#319]
We are passing options for Tangram (`scene`, `debug`, `fallbackTile`) with other options for `L.Mapzen.map`. Based on https://github.com/mapzen/mapzen.js/issues/260 , there will be more to come, so it will be better to at least wrap them as an object. 

Syntax can be like this

```
L.Mapzen.map('map', {
  apikey: 'your-api-key',
  tangramOptions: {
    debug: true, 
    scene: L.Mapzen.Refill,
    preFunction: ~~~
  }
});

```

**Comment by rfriberg** [comment]
Yes!  Is this different from #260 ?  Are you proposing just wrapping the Tangram options we currently support?  Any reason we shouldn't just jump to supporting all available Tangram options?

**Comment by hanbyul-here** [comment]
It is slightly different because MapzenJs some variables (`debug`, `fallbackTile`) for Tangram component. I am suggesting this as a pre-step to expose all available Tangram options.

**Comment by rfriberg** [comment]
Merged.  Will be released with v0.6 (later this week).
